src/main/java/net/logstash/logback/appender/AsyncDisruptorAppender.java

```diff
@@ -645,21 +605,10 @@ protected void setEventTranslator(EventTranslatorOneArg<LogEvent<Event>, Event>
         this.eventTranslator = eventTranslator;
     }
 
-    protected ScheduledExecutorService getExecutorService() {
-        return executorService;
-    }
-
```

Comment:
probably should move this method down to AbstractLogstashTcpSocketAppender to keep source compatibility if anyone has custom subclasses

Reply:
I thought about it but was wondering if it actually was a good idea. Using the executorService without proper coreSize sizing may result in unpredictable behaviour. Wouldn't be safer to not expose it and let people use their own ExecutorService should they need one ?

Reply:
That is a excellent point.  I agree.  Let's keep it encapsulated and not provide access.